etc/clang_plugin/plugin_ASTAnalyzer.cpp

```diff
@@ -850,6 +849,10 @@ class DataflowHLSPragmaHandler : public HLSPragmaAnalyzer, public HLSPragmaParse
 
                      hasModule = true;
                   }
+                  else if(callExpr->isTypeDependent())
```

Comment:
We could check for this on FD directly and skip the whole method. We could try DeclContext::isDependentContext.

Reply:
I am not sure how to implement this comment. Could you please suggest a solution?